crates/gosub_css3/src/colors.rs

```diff
@@ -97,17 +96,12 @@ impl From<&str> for RgbColor {
             return RgbColor::new(rgb.get_red(), rgb.get_green(), rgb.get_blue(), rgb.get_alpha());
         }
 
-        return get_hex_color_from_name(value).map_or(RgbColor::default(), parse_hex);
+        get_hex_color_from_name(value).map_or(RgbColor::default(), parse_hex)
     }
 }
 
 fn get_hex_color_from_name(color_name: &str) -> Option<&str> {
-    for entry in crate::colors::CSS_COLORNAMES.iter() {
-        if entry.name == color_name {
-            return Some(entry.value);
-        }
-    }
-    None
+    Some(crate::colors::CSS_COLORNAMES.iter().find(|entry| entry.name == color_name)?.value)
```

Comment:
The only thing that might be a bit stylistic unpretty is this here, the line is relatively long. But the rest looks good!

Reply:
I think it would be a bit nicer if it's separated on different lines in this case.. but we should use 120char (or even 140 ftm) line lengths..  we should really need to see if we can somehow enforce this in the linting check on CI probably..   It works quite well in go, but somehow it doesn't play nice with rust